src/main/java/seedu/command/event/EditEvent.java

```diff
@@ -0,0 +1,22 @@
+package seedu.command.event;
+
+import seedu.command.Command;
+import seedu.event.Event;
+import seedu.event.EventList;
+
+public class EditEvent extends Command {
+    private Integer index;
+    private Event newEvent;
+    private EventList eventList;
+
+    public EditEvent(Integer index, Event newEvent, EventList eventList) {
+        this.index = index;
+        this.newEvent = newEvent;
+        this.eventList = eventList;
+    }
```

Comment:
If this is newEvent, for editing date and time, can we change the variable name to newDateTime